pkgs/data/fonts/intel-one-mono/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "intel-one-mono";
+  version = "1.2.0";
+
+  src = fetchzip {
+    url = "https://github.com/intel/intel-one-mono/releases/download/V${version}/ttf.zip";
+    sha256 = "sha256-uZRRiwjBr3Cm10uh0ji65+ipSkKSKa8VOx8ZVrZOJKI=";
+    stripRoot = false;
+  };
```

Comment:
You might wan't to add a comment that this is in this case more efficient then using the fetchFromGitHub wrapper because there are more files in the release

Reply:
At this point if we want to keep trutype maybe I will switch to fetchFromGithub